cli/src/commands/init.rs

```diff
@@ -311,3 +324,11 @@ fn prompt_depfile_type(depfile: &str, mut formats: Vec<&str>) -> io::Result<Stri
 
     Ok(formats[index].to_owned())
 }
+
+/// Get repository URL from current directory's git info.
+fn git_repository_url() -> Option<String> {
+    let repository = Repository::open(".").ok()?;
+    let remote = repository.find_remote("origin").ok()?;
+    let url = remote.url()?;
+    url.starts_with("http").then(|| url.into())
```

Comment:
What about trimming any trailing `.git` suffixes? URLs with those appear to redirect to the page without the suffix, but it might still look odd to include it in output. It appears the Phylum GitHub app and the integrations record repo URLs without the `.git` suffix.

```
# After cloning the `phylum-ci` repo with the HTTPS URL...
❯ git remote -v
origin	https://github.com/phylum-dev/phylum-ci.git (fetch)
origin	https://github.com/phylum-dev/phylum-ci.git (push)

❯ /Users/maxrake/dev/phylum/localdev/cli/target/debug/phylum init
⚠️  Workspace is already linked to a Phylum project
Overwrite existing project configuration? yes

Project Name [default: phylum-ci]:

[ENTER] Confirm
Project Group: integrations


Use "https://github.com/phylum-dev/phylum-ci.git" as repository URL? yes
[SPACE] Select  [ENTER] Confirm
Select your project's dependency files: ./poetry.lock

✅ Successfully created project configuration

❯ phylum project list -g integrations
Project Name   Project ID                            Repository URL
locksmith      f4c7fdcc-0a4b-49c7-8a37-7b4635b12662
birdcage       2136f54e-5a87-4864-ac01-aab58e725f5d
TestADO        13c5532b-e2e0-4984-951c-97af1a238321  https://github.com/maxrake/TestADO
TestJenkinsGH  20a52a4b-fbd0-47d8-b9c0-063b1098bf14  https://github.com/maxrake/TestJenkinsGH
phylum-ci      08f8466e-16d6-430a-9006-4e1dfcd684f7  https://github.com/phylum-dev/phylum-ci.git
cli            d84b0bb3-fb64-43cb-a473-dc96b54db662
rubocop        f884cfed-195c-41e5-bc75-82aae1d74f6b
TestGL         1bf35e35-c1b1-4fab-adb6-b64e43486aa5
TestBB         935bdc06-13e8-4ead-a2b7-e679c69b8970  https://bitbucket.org/phylum-maxrake/testbb
TestGHA        3c0baddd-ac47-4417-8c50-97aedae0c96f
TestGitLab     f18e0045-7e72-42f8-9f03-43d534761ca1  https://gitlab.com/maxrake/gltestrepo
```

Reply:
I have no interest in performing even more heuristics than just eliminating everything without a leading `http`. If this data isn't clean enough for usage, then we should scrap this feature entirely. Randomly modifying URLs and hoping that they still work is not a proper way to build repository URLs.

Reply:
Roger that...I won't hold it up then.